client/my-sites/media-library/content.jsx

```diff
@@ -135,6 +135,9 @@ class MediaLibraryContent extends React.Component {
 						i18nOptions
 					);
 					break;
+				case MediaValidationErrors.SERVICE_AUTH_FAILED:
```

Comment:
Tried to follow where this message is first thrown. As far as I understood, it's dispatched [here](https://github.com/Automattic/wp-calypso/blob/master/client/lib/media/actions.js#L57) as the result of a server response. It looks like it might be thrown by **any** service, not only the google photos one. The name being not specific to Google also leads me to believe that might be the case. If that's true, shouldn't we take into consideraction the service who threw it before showing the notice?

Reply:
Added the check.  I had a more complicated notice the first time round, and on a relook it made sense to shuffle stuff around as you suggested below, so I did that too.

Reply:
:heartbeat: